schema/course.schema.json

```diff
@@ -8,6 +8,43 @@
     },
     "with": {
       "properties": {
+        "_globals": {
+          "type": "object",
+          "default": {},
+          "properties": {
+            "_extensions": {
+              "type": "object",
+              "default": {},
+              "properties": {
+                "_bookmarking": {
+                  "type": "object",
+                  "title": "Bookmarking",
+                  "default": {},
+                  "properties": {
+                    "resumeButtonLabel": {
+                      "type": "string",
+                      "title": "Resume button label",
+                      "description": "Label for ubiquitous resume button.",
+                      "default": "Resume",
+                      "_adapt": {
+                        "translatable": true
+                      }
+                    },
+                    "resumeButtonAriaLabel": {
+                      "type": "string",
+                      "title": "Resume ARIA label",
+                      "description": "ARIA label for ubiquitous resume button.",
```

Comment:
Description text isn't necessary here. Use of the word ubiquitous is perhaps a little much for non-English speakers.
```suggestion
```